frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -68,6 +75,8 @@ local symbol_prefix = {
         frontlight = "☼",
         mem_usage = "",
         wifi_status = "",
+        book_title = "✎",
+        book_chapter = "☝",
```

Comment:
Might find nicer icons in the nerdfont @NiLuJe recently added (he uses some of them for the battery icon). https://github.com/ryanoasis/nerd-fonts
Or let these (or just one?) without icons/prefix. They should be sufficiently different from all the other items (which are mostly numbers) so there's no doubt about what they are.

Reply:
I'm pretty sure there's at least an (open) book icon, yeah.

But I think I agree with @poire-z that these don't necessarily need a prefix/icon ;).

Reply:
+1 There's no need for an icon just like there's no need for an icon with the date or something.

Reply:
c.f., a search fro `book` on https://www.nerdfonts.com/cheat-sheet

(The codepoint won't match with our version, ping me if you find something you like, I'll pull the CP for you).

Reply:
I also like idea to not add icons/prefix to title and chapter.